mysqld-mixin/Makefile

```diff
@@ -0,0 +1,21 @@
+JSONNET_FMT := jsonnetfmt -n 2 --max-blank-lines 2 --string-style s --comment-style s
+
+all: fmt lint build clean
```

Comment:
better to add `default: build`. 

Currently it defaults to `all`.  (meaning it builds and removes the dashboards as well)
idea is to run `build` step when I just type `make` :) more intuitive!